pytorch_lightning/utilities/imports.py

```diff
@@ -68,6 +68,8 @@ def _compare_version(package: str, op, version) -> bool:
 _TORCH_GREATER_EQUAL_1_8 = _compare_version("torch", operator.ge, "1.8.0")
 _TORCH_GREATER_EQUAL_1_8_1 = _compare_version("torch", operator.ge, "1.8.1")
 _TORCH_GREATER_EQUAL_1_9 = _compare_version("torch", operator.ge, "1.9.0")
+_TORCH_GREATER_EQUAL_1_10 = _compare_version("torch", operator.ge, "1.10.0dev")
```

Comment:
@Borda is this correct? I had to add the dev in for the equality to work correctly.

Reply:
I think without dev as there is not there and also we want to have it valid even after release (this is related to PL 1.10 not only dev)